### PR TITLE
added customtemplates event & eventTitle

### PR DESCRIPTION
**Pull request by sch3bal**
Hi
 
I added these two lines so the Custom-Templates for Event and EventTitle works like in the original Repo.

### CODE REVIEW

File: src/app/modules/calendar-week-hours-view/calendar-week-hours-view.component.ts [hunk 1]
```diff
@@ -445,7 +447,7 @@ export class CalendarWeekHoursViewComponent implements OnChanges, OnInit, OnDest
             excluded: this.excludeDays,
             precision: this.precision,
             absolutePositionedEvents: true
-        });
+        }).eventRows;
```
Comment (diegofsza) [inline src/app/modules/calendar-week-hours-view/calendar-week-hours-view.component.ts]: Is this modification really meant to be there? The compiler complains about this: Property 'eventRows' does not exist on type 'WeekViewEventRow[]'

Comment (sch3bal) [reply src/app/modules/calendar-week-hours-view/calendar-week-hours-view.component.ts]: Actually yes...
I get this error when I remove this line:
Type 'WeekView' is not assignable to type 'WeekViewEventRow[]'

Comment (diegofsza) [reply src/app/modules/calendar-week-hours-view/calendar-week-hours-view.component.ts]: Ok. It seems that the `getWeekView` method in `CalendarUtils` changed its signature between versions 0.22.3 and current (0.23.2). I think we need to update the peerDependency version for angular-calendar to make this change work.

**Comment by mohanrex** [comment]
Hi,

Will it be merged? If you don't want the modification atleast we could change it and mege it right.

**Comment by diegofsza** [comment]
Hi, I think that the peer dependency on the package.json should be changed to be consistent. Maybe a comment about versions compatibility in the readme could be helpful, but not entirely necessary though.

**Comment by rodrigoreal** [comment]
Hey @diegofsza 
Will it be merged? I really need that feature.

**Comment by diegofsza** [comment]
This is included in version 1.0.1.

**Pull request merged**